src/connector/mysql.rs

```diff
@@ -391,4 +391,52 @@ mod tests {
         let err = res.unwrap_err();
         assert!(matches!(err.kind(), ErrorKind::AuthenticationFailed { user } if user == "WRONG"));
     }
+
+    #[tokio::test]
+    async fn should_pick_up_partially_failed_raw_cmd_scripts() {
+        let conn = Quaint::new(&CONN_STR).await.unwrap();
+
+        let result = conn.raw_cmd("SELECT YOLO; SELECT 1;").await;
+
+        assert!(result.is_err());
+
+        let result = conn.raw_cmd("SELECT 1; SELECT YOLO;").await;
+
+        assert!(result.is_err());
+    }
+
+    #[tokio::test]
+    async fn should_execute_multi_statement_queries_with_raw_cmd() {
+        let conn = Quaint::new(&CONN_STR).await.unwrap();
+
+        conn.raw_cmd("DROP DATABASE `raw_cmd_test`;").await.unwrap();
+        conn.raw_cmd("CREATE DATABASE `raw_cmd_test`;").await.unwrap();
+
+        let conn = Quaint::new(&CONN_STR.replace("dbname", "raw_cmd_test")).await.unwrap();
+
+        conn.raw_cmd(
+            "
+            CREATE TABLE `raw_cmd_test`.`testtable` (id INTEGER PRIMARY KEY);
```

Comment:
temporary table?

Reply:
This is crashing. Let's discuss this later, but I'm getting pessimistic about fixing this.